documentation/index.rst

```diff
@@ -11,6 +11,10 @@ developing for Linux. **kw** is composed of different software components
 unified in a single interface. After installation, ``kw`` commands become
 available in the command line interface.
 
+kw project is available at https://github.com/kworkflow/kworkflow, and we also
+have a blog post which you can access at https://blog.kworkflow.org/. For more
```

Comment:
I think it would be better if it were either "we also have a blog which..." or "we also have blog posts which..."